cli/run_util.go

```diff
@@ -166,10 +166,16 @@ func (r WorkflowRunner) activateAndRunSteps(
 				failedStepRunResult := currentBuildRunResult.FailedSteps[0]
 				failedStepEnvs := bitrise.FailedStepEnvs(failedStepRunResult)
 				*environments = append(*environments, failedStepEnvs...)
+				if currentStepBundleUUID != "" {
```

Comment:
Can we also add a test case for verifying the failed step env vars within a step bundle?